data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/networkhandlers/PeersHandler.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.beaconrestapi.networkhandlers;
+
+import tech.pegasys.artemis.beaconrestapi.handlerinterfaces.BeaconRestApiHandler;
+import tech.pegasys.artemis.networking.p2p.JvmLibP2PNetwork;
+import tech.pegasys.artemis.networking.p2p.api.P2PNetwork;
+import tech.pegasys.artemis.provider.JsonProvider;
+
+public class PeersHandler implements BeaconRestApiHandler {
+
+  private final P2PNetwork network;
+  private final boolean isLibP2P;
+
+  public PeersHandler(P2PNetwork network, boolean isLibP2P) {
+    this.network = network;
+    this.isLibP2P = isLibP2P;
+  }
+
+  @Override
+  public String getPath() {
+    return "/network/peers";
+  }
+
+  @Override
+  public String handleRequest(RequestParams param) {
+    if (isLibP2P) {
```

Comment:
one last nit: we should actually just replace the `isLibP2P` flags being passed around and do the `network instanced JvmLibP2PNetwork` here (and in the other places we're checking the `isLibP2P` flag).  Avoids the potential for `isLibP2P` and the network type to wind up out of sync and reduces the code a bit.

Reply:
Gotcha. On it.

Reply:
Done.